app/controllers/settings/featured_tags_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Settings::FeaturedTagsController < Settings::BaseController
+  layout 'admin'
+
+  before_action :authenticate_user!
+  before_action :set_featured_tags, only: :index
+  before_action :set_featured_tag, except: [:index, :create]
+
+  def index
+    @featured_tag   = FeaturedTag.new
+    @most_used_tags = Tag.most_used(current_account).where.not(id: @featured_tags.map(&:id)).limit(10)
```

Comment:
Probably want `@featured_tags.pluck(:id)` to avoid pulling all of those records into memory.

Reply:
Also, depending on `Tag`'s database size, you probably want to do `WHERE NOT EXIST IN (SELECT 1 FROM "featured_tags" WHERE "featured_tags"."id" = "tags"."id")`, see: https://sqlperformance.com/2012/12/t-sql-queries/left-anti-semi-join

Reply:
`@featured_tags` is already loaded and it's expected to be a small array.

Reply:
Can you point to where it's already loaded? I don't believe it is.